diff -p -up extreme-tuxracer-0.5beta/src/game_config.cpp.defaultopt extreme-tuxracer-0.5beta/src/game_config.cpp
--- extreme-tuxracer-0.5beta/src/game_config.cpp.defaultopt	2008-06-29 17:20:18.000000000 +0200
+++ extreme-tuxracer-0.5beta/src/game_config.cpp	2009-03-15 09:47:34.000000000 +0100
@@ -538,7 +538,7 @@ void init_game_configuration()
 	"# at the cost of a few Hz." );
 
     INIT_PARAM_BOOL( 
-	display_fps, false,
+	display_fps, true,
 	"# Set this to true to display the current framerate in Hz." );
 
     INIT_PARAM_BOOL( 
@@ -547,11 +547,11 @@ void init_game_configuration()
 	"# the course percentage." );
 
     INIT_PARAM_INT( 
-	x_resolution, 800,
+	x_resolution, 800,
 	"# The horizontal size of the Tux Racer window" );
 
     INIT_PARAM_INT( 
-	y_resolution, 600,
+	y_resolution, 600,
 	"# The vertical size of the Tux Racer window" );
 
 	INIT_PARAM_BOOL( 
@@ -587,7 +587,7 @@ void init_game_configuration()
         "# decreasing this number, at the cost of lower image quality." );
 
     INIT_PARAM_BOOL( 
-	fullscreen, true,
+	fullscreen, false,
 	"# If true then the game will run in full-screen mode." );
 
     INIT_PARAM_INT( 
@@ -833,7 +833,7 @@ void init_game_configuration()
 	"# This option would improve the performance." );
 	
 	INIT_PARAM_STRING( 
-	ui_language, "en_GB" ,
+	ui_language, "en" ,
 	"# set the language for the ui"
 	);
 	
